src/common/ErrorMsg.js

```diff
@@ -27,7 +29,9 @@ export default class ErrorMsg extends React.PureComponent {
 
     return (
       <View style={styles.field}>
-        <Text style={styles.error}>{error}</Text>
+        <Label style={styles.error}>
+          {error}
```

Comment:
Label not having text ultimately FormattedMessage not having ID throwing an error